pages.es/common/immich-cli.md

```diff
@@ -0,0 +1,29 @@
+# immich-cli
+
+> Immich tiene una interfaz de línea de comandos (CLI) que le permite realizar ciertas acciones desde la línea de comandos.
+> Véase también: `immich-go`.
+> Más información: <https://immich.app/docs/features/command-line-interface/>.
+
+- Autenticarse en el servidor de Immich:
+
+`immich login {{http://192.168.1.69:2283/api}} {{HFEJ38DNSDUEG}}`
```

Comment:
```suggestion
`immich login {{http://192.168.1.69:2283/api}} {{HFEJ38DNSDUEG}}`
```

Make sure you are not sharing personal or sensitive information in the pages.

Reply:
Yes, it's an official example, "/api" should not be removed.

Reply:
I have re-added the `api` part of the URL